inline.h

```diff
@@ -1177,6 +1177,20 @@ Perl_rpp_invoke_xs(pTHX_ CV *cv)
 }
 
 
+/* for SvCANEXISTDELETE() macro in pp.h */
+PERL_STATIC_INLINE bool
+Perl_sv_can_existdelete(pTHX_ SV *sv)
+{
+    /* Anything without tie magic is fine */
+    MAGIC *mg;
+    if(!SvRMAGICAL(sv) || !(mg = mg_find((const SV *)sv, PERL_MAGIC_tied)))
```

Comment:
Why the `(const SV *)` cast?

Reply:
A good question, I admit I just copypasted that from the original macro body. Turns out not to be needed; probably only required for the same reason the outer macro has it.